internal/factsengine/gatherers/sbd.go

```diff
@@ -0,0 +1,52 @@
+package gatherers
+
+import (
+	"github.com/joho/godotenv"
+	log "github.com/sirupsen/logrus"
+	"github.com/trento-project/agent/internal/cluster"
+)
+
+const (
+	SBDConfigGathererName = "sbd_config"
+	UndefinedSBDConfig    = "trento.checks.sbd.errors.undefined_configuration"
+)
+
+type SBDGatherer struct {
+	configFile string
+}
+
+func NewSBDGathererWithDefaultConfig() *SBDGatherer {
+	return NewSBDGatherer(cluster.SBDConfigPath)
+}
+
+func NewSBDGatherer(configFile string) *SBDGatherer {
+	return &SBDGatherer{
+		configFile,
+	}
+}
+
+func (g *SBDGatherer) Gather(factsRequests []FactRequest) ([]Fact, error) {
+	gatheredFacts := []Fact{}
+	log.Infof("Starting SBD Facts gathering")
+
+	conf, err := godotenv.Read(g.configFile)
```

Comment:
We could use the `utils.FindMatches` function here, which does exactly the same, and we avoid other dependency.
It could be:
```
conf := utils.FindMatches(`(?m)^(\w+)=(\S[^#\s]*)`, fileContent)
```

Reply:
In fact, as you already import `cluster`, it has a method named `getSBDConfig`, which does already the same

Reply:
This is interesting. I absolutely didn't see/remember that function :sweat_smile: 

Anyway, could you help me understand the issue with adding the https://github.com/joho/godotenv dependency?

A middle ground that I would find useful would be the other way around:
refactoring what we currently do in `cluster.getSBDConfig` by using `godotenv`, export that funcion and use it in the gatherer.

Less code, less things to test and maintain :stuck_out_tongue: 
Anyway the library looks quite stable and suitable for the kind of file we are processing.

What do you think?